verilog/analysis/checkers/parameter_name_style_rule.cc

```diff
@@ -68,9 +68,9 @@ void ParameterNameStyleRule::HandleSymbol(const verible::Symbol& symbol,
 
     const verible::TokenInfo* param_name_token = nullptr;
     if (IsParamTypeDeclaration(symbol))
-      param_name_token = &GetSymbolIdentifierFromParamDeclaration(symbol);
-    else
-      param_name_token = &GetParameterNameToken(symbol);
+      return;
```

Comment:
With this change, update the documentation for this class, along with its message strings saying that this is applicable to non-type parameters.

Reply:
Added, thanks!